src/app/users/create-user/passwords-must-match.validator.ts

```diff
@@ -0,0 +1,17 @@
+/** Angular Imports */
```

Comment:
Don't create a new validator for this. We have an existing validator here: https://github.com/openMF/web-app/blob/master/src/app/login/reset-password/confirm-password.validator.ts
You can use this instead. I know this is currently in the login folder but later on, we can move this into the shared folder and make it more generic. (after GCI)